Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,285 @@
+(Demos\HandInteraction\Scenes\HandInteractionDemos.unity) for a demonstration of using the LaunchUri script
+to start an external application.
+
+#### SceneSystem
+
+MRTK 2.0.0 has added the [Scene System](SceneSystem/SceneSystemGettingStarted.md) to help with
+applications that contain more than once scene.
+
+#### Input Animation Recording
+ 
+MRTK 2.0.0 features a [recording system](InputSimulation/InputAnimationRecording.md) by which head movement and hand tracking
+data can be stored in animation files. The recorded data can then be played back using the [input simulation system](InputSimulation/InputSimulationService.md).
+
+#### Hand Menu Graduated from Experimental
+
+The HandConstraint and HandConstraintPalmUp solvers are now an official feature (no longer experimental) and have improved documentation.
+
+The HandConstraintPalmUp solver now has a toggle to enforce the hand’s fingers are coplanar before activating.
+
+#### UX Controls
+
+The following UX controls have been added and/or improved in version 2.0.0. In addition, those in the
+following list can now be instantiated and configured from code.
+
+- BoundingBox
+- ManipulationHandler
+- HandInteractionPanZoom
+- Interactable (basic features)
+
+**HoloLens 2 Button**
+
+- Improved many visual details to match the HoloLens 2 shell including
+    - Compressing visuals
+    - Far interaction support
+    - Focus highlight
+    - Shader effects
+- HoloLens 2 style Round Button has been added
+
+**Fingertip Cursor**
+
+The fingertip cursor has been updated to better match the HoloLens 2 shell.
+
+**BoundingBox**
+
+- Improvements
+    - Normalized the handle asset size and scaling logic
+    - The handle asset is now sized to 1-meter
+    - Default values and examples are updated
+- New features
+    - Animated Handle by Proximity
+    - Match the HoloLens 2 shell behavior
+    - You can now make the handles appear only when your hand is close to them
+- New example scene
+    - The BoundingBoxExample scene shows various types of configurations
+
+Please refer to the [Bounding Box](README_BoundingBox.md) documentation for more details.
+
+**Radial Solver**
+
+There have been improvements on vertical positioning. Check ‘Use Fixed Vertical Position’ to lock the vertical movement to achieve shell-style tag-along behavior. You can see the example of lazy-following tag-along behavior in the ‘ToggleFeaturesPanel’ prefab .
+
+**Clipping Example**
+
+The ClippingExamples scene, demonstates using the MRTK Standard Shader’s new clipping feature.
+
+**Slate**
+
+- Improved
+    - Usability of slates by adding the shadow based on the finger proximity
+    - ‘Follow Me’ behavior to match the HoloLens 2 shell behavior, using Radial Solver.
+- Fixed
+    - Border thickness issue fixed on flattend Bounding Box
+
+#### MRTK Standard Shader
+
+The [MRTK Standard Shader](README_MRTKStandardShader) now supports Unity's Lightweight Scriptable render pipeline.
```

Comment:
Here are a few more MRTK Standard Shader features that might be nice to point out?

•	A  warning and "Fix Now" button is now displayed on materials which are in a project that has depth buffer sharing enabled and do not write depth.
•	Proximity lights now have per material color overrides and a "Subtractive Light" option to mimic proximity shadows.
•	Clipping primitives can now be toggled on/off at runtime without requiring shader permutations to be known at (player) build time.
•	Additive and transparent materials are now clipped correctly. A new property for non-opaque or cutout materials called “Blended Clipping Width” allows developers to set the width of the clip blend region.
•	Lighting of back face triangles is now correct.

Reply:
added. will be in the next commit